### PR TITLE
Adds drinks we have sprites for.

### PR DESCRIPTION
**Pull request by Mitchs98**
## What Does This PR Do
Just adds in a few drinks that we had sprites for in the code, and one that we had a recipe for but no sprite. Sprite for the Kahlua Vodka done by Pewtershmitz on Discord. You can uh, ignore the branch name as it did start out as a refactor >_>.

Code_vAicEFFdcr
Code_WMufgLaDGL

## Why It's Good For The Game
Stuff already exists, figure there's no reason to not add it into the game. Otherwise pretty useless.

## Images of changes

dreammaker_5WKTO8nZ4J

## Changelog
:cl:
add: A few new drinks are now able to be made. The Hearty Punch, Fetching Fizz, Arnold Palmer, and Kahlua Vodka.
/:cl:

### CODE REVIEW

**Comment by Krysonism** [comment]
Paradise players should be aware of that fetching fizz and hearty punch have special effects on /tg/ you should consider porting in some form.

Most bartender do not bother making hearty punch on /tg/, but you feel like a badass when you manage to save one of your patrons from death by feeding them some hearty punch.

**Comment by Mitchs98** [comment]
I can try and port said effects if the maints believe they're fine for the drinks to have, I personally had no idea they did anything.

**Comment by Mitchs98** [comment]
I uh...didn't mean to do that :D

File: code/modules/reagents/chemistry/reagents/drinks.dm [hunk 1]
```diff
@@ -510,3 +510,43 @@
...
+	
+/datum/reagent/consumable/drink/kahluaVodka
+	name = "Kahlua Vodka"
+	id = "kahluaVodka"
+	description = "For the Space Russian on the go."
+	color = "#664300" // rgb: 102, 67, 0
+	drink_icon = "kahluaVodka"
+	drink_name = "Kahlua Vodka"
+	description =  "For the Space Russian on the go."
+	taste_description = "sweet vodka"
```
Comment (farie82) [inline code/modules/reagents/chemistry/reagents/drinks.dm:552]: missing an end of line here.

File: code/modules/reagents/chemistry/reagents/drinks.dm [hunk 2]
```diff
@@ -510,3 +510,43 @@
...
+	name = "Hearty Punch"
+	id = "heartypunch"
+	description = "This drink will give you the punch in the arm you need to keep going."
+	color = "#664300" // rgb: 102, 67, 0
+	drink_icon = "hearty_punch"
+	drink_name = "Hearty Punch"
+	description = "This drink will give you the punch in the arm you need to keep going."
+	taste_description = "encouragement"
+	
+/datum/reagent/consumable/drink/kahluaVodka
```
Comment (farie82) [inline code/modules/reagents/chemistry/reagents/drinks.dm:544]: You can't create this one currently in your PR.
image
The recipe already exists but just creates kahlua

**Comment by Fox-McCloud** [comment]
I'd just implement what TG has---where these sprites came from.

**Comment by Fox-McCloud** [comment]
Any plans on updating? Very stale; no response after a week.

**Comment by Mitchs98** [comment]
Yeah sorry. I'll get to the stuff Farie pointed out. And if you'd prefer TG's implementation I can look into that too.

**Comment by Fox-McCloud** [comment]
Yes, please.

**Comment by Mitchs98** [comment]
Yeaah I murdered the hell out of this trying to update the branch. I have no idea why, I magically break simple things. Closing and re-opening on a new branch once I get the requested changes sorted.

**Pull request closed without merging**